config/formats.js

```diff
@@ -605,6 +605,18 @@ let Formats = [
 				}
 			}
 		},
+		// @ts-ignore
+		onHitPriority: 999,
+		onHit: function (target, source, move) {
+			// @ts-ignore
+			if (move.category !== 'Status' && source.forte && source.forte.onHit) this.singleEvent('Hit', source.forte, {}, target, source, move);
+		},
+		// @ts-ignore
+		onAfterSubDamagePriority: 999,
+		onAfterSubDamage: function (damage, target, source, move) {
+			// @ts-ignore
```

Comment:
instead of ts-ignoring everything, you can just declare these under ``FormatsData`` in globals.ts

Reply:
Actually it'd be cool if event priorities were in the `EventMethods` interface or something, but that could be its own pull request

Reply:
you can still just put them under FormatsData for now.